web/src/components/snapshots/AppSnapshotRestore.jsx

```diff
@@ -333,8 +326,7 @@ class AppSnapshotRestore extends Component {
     }
 
     if (restoreCompleted && hasNoErrorsOrWarnings) {
-      this.logOutUser();
-      this.props.history.push("/restore-completed");
+      Utilities.logoutUser(null, { snapshotRestore: true });
```

Comment:
Previously this was using it's own logout command that did basically the same thing as the utility. I got rid of that and swapped in the utility.